pandas/core/arrays/categorical.py

```diff
@@ -1568,6 +1574,11 @@ def sort_values(self, inplace=False, ascending=True, na_position="last"):
         na_position : {'first', 'last'} (optional, default='last')
             'first' puts NaNs at the beginning
             'last' puts NaNs at the end
+        key : callable, default None
+            If not None, apply the key function to every value before
+            sorting. Identical to key argument in built-in sorted function.
```

Comment:
Is this description correct? Is `key` applied to every value or to the array?

Reply:
Did you change the behavior for `Categorical`? This looks like the way I implemented it originally.

Reply:
I don't recall.

Reply:
It has the new behavior now, we just hadn't updated the documentation.